perf_test/blas/blas3/KokkosBlas3_gemm_perf_test.hpp

```diff
@@ -1116,6 +1116,106 @@ void __do_gemm_parallel_experiment6(options_t options, gemm_args_t gemm_args) {
   return;
 }
 
+template <class MemberType, class SimdViewType, class TransAType, class TransBType,
+          class BlockingType>
+class parallel_batched_gemm_experiment7 {
+ private:
+  SimdViewType &A, &B, &C;
+  gemm_args_t gemm_args;
+
+ public:
+  parallel_batched_gemm_experiment7(SimdViewType &_A, SimdViewType &_B,
+                                    SimdViewType &_C, gemm_args_t _gemm_args)
+      : A(_A), B(_B), C(_C), gemm_args(_gemm_args) {}
+
+  KOKKOS_INLINE_FUNCTION
+  void operator()(const MemberType &member) const {
+    auto i = member.league_rank();
+    Kokkos::parallel_for(Kokkos::ThreadVectorRange(member, A.extent(0)),[&](const int &vector_lane) {
+	auto svA = Kokkos::subview(A, vector_lane, Kokkos::ALL(), Kokkos::ALL(), i);
+	auto svB = Kokkos::subview(B, vector_lane, Kokkos::ALL(), Kokkos::ALL(), i);
+	auto svC = Kokkos::subview(C, vector_lane, Kokkos::ALL(), Kokkos::ALL(), i);
+
+	KokkosBatched::TeamGemm<MemberType, TransAType, TransBType, BlockingType>::invoke(member, gemm_args.alpha, svA, svB, gemm_args.beta, svC);
+   });
+  }
+};
+
+template <class TransAType, class TransBType, class BlockingType,
+          class device_type>
+void __do_gemm_parallel_experiment7(options_t options, gemm_args_t gemm_args) {
```

Comment:
@e10harvey Could you please change this experiment to an actual test name? Can this test work with both ```Blocked``` and ```Unblocked```?

Reply:
Yes to both.